srcpkgs/rizin/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'rizin'
+pkgname=rizin
+version=0.3.4
+revision=1
+archs="i686 x86_64"
+wrksrc="${pkgname}-v${version}"
+build_style=meson
+configure_args="-D use_sys_capstone=enabled -D use_capstone_version=v4
+ -D use_sys_magic=enabled -D use_sys_libzip=enabled -D use_sys_zlib=enabled
+ -D use_sys_lz4=enabled -D use_sys_xxhash=enabled -D use_sys_openssl=enabled
+ -D use_sys_tree_sitter=enabled -D use_libuv=true -D use_webui=true -D local=disabled"
+hostmakedepends="pkg-config"
+makedepends="capstone-devel libzip-devel zlib-devel liblz4-devel xxHash-devel libuv-devel
+ tree-sitter-devel file-devel openssl-devel"
+short_desc="UNIX-like reverse engineering framework and command-line toolset"
+maintainer="Urs Schulz <voidpkgs@ursschulz.de>"
+license="LGPL-3.0-only"
+homepage="https://github.com/rizinorg/rizin"
+distfiles="https://github.com/rizinorg/${pkgname}/releases/download/v${version}/${pkgname}-src-v${version}.tar.xz"
+checksum=eea49b396387c09d19705aab02a617cdb15682fca67f101ff2b27eef94a710e9
+
+subpackages="rizin-devel"
+
+
+rizin-devel_package() {
+	depends="${makedepends} rizin>=${version}_${revision}"
+	short_desc="${short_desc} - development files"
+	pkg_install() {
+		vmove usr/include/librz
+		vmove usr/lib/pkgconfig
+	}
```

Comment:
```suggestion
	pkg_install() {
		vmove "usr/lib/*.so"
		vmove usr/include/librz
		vmove usr/lib/pkgconfig
	}
```

Reply:
The `.so` files definitely DO NOT go into the `-devel` package.

Reply:
They are just symlinks to .so.$version, we don't need them in the main package, we only need the real versioned libraries.